tools/downloader/src/open_model_zoo/model_tools/_common.py

```diff
@@ -675,3 +690,21 @@ def quote_arg_windows(arg):
 
 def command_string(args):
     return ' '.join(map(quote_arg, args))
+
+@contextlib.contextmanager
+def telemetry_session(tool):
+    telemetry = Telemetry('Model Downloader', tid='UA-194864834-1')
```

Comment:
Hasn't the first argument to `Telemetry` previously been different for each tool?

Reply:
Yes, fixed